plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -312,6 +312,7 @@ def create_range_param_using_interval_override(override: Override) -> Dict[str,
         "name": key,
         "type": "range",
         "bounds": [value.start, value.end],
+        "log_scale": "log" in value.tags,
```

Comment:
I would slightly prefer to use `"log_scale": "log_scale" in value.tags` instead of `"log_scale": "log" in value.tags` here. The motivation is:
  1) that way the API would be consistent between using the command line vs using yaml files, and
  2) there would be less chance for users to be confused by thinking that they are e.g. taking the logarithm of the values (which is not the case).

What do you think, @SiddhantRanade?

Reply:
Please disregard my previous comment -- I hadn't seen pixelb's comment above. It seems that the `log` tag is already being used by other sweepers (nevergrad and optuna).